### ISSUE TITLE
gribdoctor

### ISSUE THREAD

**Issue by ian29** [mapbox/preprocessorcerer#8]
@rclark @dnomadb i wonder if down the line it makes sense to incorporate [our colleague Dr. Grib](https://github.com/mapbox/grib-doctor) for easy weather uploads.

**Comment by rclark** [comment]
@ian29 we chatted about this but I don't quite remember the outcome. Would the idea be that you'd upload a grib and then unpacker would make tiles from it? That would mean the preprocessorcery would be conversion to TIFF?

**Comment by ian29** [comment]
Would the idea be that you'd upload a grib and then we would make tiles from it? That would mean the preprocessorcery would be conversion to TIFF?

yes and yes, assuming this is generally how preprocessorcerer works?

**Comment by rclark** [comment]
Yeah my confusion comes from looking briefly at https://github.com/mapbox/grib-doctor -- it sounds like its about aggregating a set of gribs into a tiff, while our upload pipeline deals with one file at a time. 

If this is a one-at-a-time kind of operation you're thinking of, then yeah -- we could do this.

**Comment by ian29** [comment]
@rclark oh yeah. to clarify gribdoctor *can* combined (aka smoosh) many gribs into 1 tif, but also supports 1:1 grib->tif transformation.
